arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/stream/TerminalOps.kt

```diff
@@ -84,14 +97,29 @@ suspend fun <O> Stream<O>.lastOrNull(): O? =
   foldChunks<O, O?>(null) { acc, c -> c.lastOrNull() ?: acc }
 
 /**
- * Runs all the effects of this [Stream], raising a [NoSuchElementException] if the stream emitted no values,
+ * Runs all the effects of this [Stream], raising a [NoSuchElementException], by default, if the stream emitted no values,
  * and returning the last value emitted otherwise.
  *
  * This a terminal operator, meaning this functions `suspend`s until the [Stream] finishes.
  * If any errors are raised while streaming, it's thrown from this `suspend` scope.
+ *
+ * It's possible to include an alternative [FailStrategy] which describes how to fail in case of not finding the desired element.
+ * //sampleStart
```

Comment:
```suggestion
 * It's possible to include an alternative [FailStrategy] which describes how to fail in case of not finding the desired element.
 *
 * ```kotlin:ank:playground
 * //sampleStart
```